mpitests/test_probs.py

```diff
@@ -43,7 +47,7 @@ def apply_probs_nonparam(tol, operation, GateWires, Wires, C_DTYPE):
     dev_cpu = qml.device("lightning.qubit", wires=num_wires, c_dtype=C_DTYPE)
     dev_mpi = qml.device(device_name, wires=num_wires, mpi=True, c_dtype=C_DTYPE)
 
-    state_vector = create_random_init_state(num_wires, dev_mpi.R_DTYPE)
+    state_vector = create_random_init_state(num_wires, dev_mpi.c_dtype)
```

Comment:
Is the API changed to now take complex types, rather than the real-type?
Also, just wondering why the UPPER->lower convention was modified?

Reply:
> Is the API changed to now take complex types, rather than the real-type?

yes, we only define complex types. Do you suggest adding the real type definition? 

> Also, just wondering why the UPPER->lower convention was modified?

I am not sure, but since I started the migration this was the convention :thinking: 
I can change it. What do you think?